### PR TITLE
Fix bug when having more than three product image

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Images are not displayed when eeditting product if they exceed three
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-2982
| How to test?  | Add a product from the BO and add more than three image for this product, try to add a legend to images => all images are displayed or a "View more" link will be shown

### CODE REVIEW

File: admin-dev/themes/default/public/theme.css [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+@font-face{font-family:Material Icons;font-style:normal;font-weight:400;src:url(e79bfd88537def476913f3ed52f4f4b3.eot);src:local("Material Icons"),local("MaterialIcons-Regular"),url(570eb83859dc23dd0eec423a49e147fe.woff2) format("woff2"),url(012cf6a10129e2275d79d6adac7f3b02.woff) format("woff"),url(a37b0c01c0baf1888ca812cc0508f6e2.ttf) format("truetype")}.material-icons{font-family:Material Icons;font-weight:400;font-style:normal;font-size:24px;display:inline-block;line-height:1;text-transform:none;letter-spacing:normal;word-wrap:normal;white-space:nowrap;direction:ltr;-webkit-font-smoothing:antialiased;text-rendering:optimizeLegibility;-moz-osx-font-smoothing:grayscale;font-feature-settings:'liga'}
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/public/theme.css]: I guess this change is not related to the PR and can be removed?

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by vincentbz** [comment]
Tested on Chrome and FF and ok, thanks !

**Comment by Quetzacoalt91** [comment]
Thank you @vincentbz & @fatmaBouchekoua
